obs/obs.go

```diff
@@ -0,0 +1,120 @@
+package main
+
+import (
+	"encoding/xml"
+	"fmt"
+	"os"
+)
+
+type Project struct {
+	XMLName      xml.Name     `json:"project" xml:"project"`
+	Name         string       `json:"name" xml:"name,attr"`
+	Title        string       `json:"title" xml:"title,omitempty"`
+	Description  string       `json:"description" xml:"description,omitempty"`
+	URL          *string      `json:"url" xml:"url,omitempty"`
+	Persons      []Person     `json:"persons" xml:"person"`
+	Repositories []Repository `json:"repositories" xml:"repository"`
+	Build        *Build       `json:"build" xml:"build,omitempty"`
+	Publish      *Publish     `json:"publish" xml:"publish,omitempty"`
+	DebugInfo    *DebugInfo   `json:"debugInfo" xml:"debuginfo,omitempty"`
+	UseForBuild  *UseForBuild `json:"useForBuild" xml:"useforbuild,omitempty"`
+}
+
+type Client struct {
+	Username string
+	Password string
+	ApiURL   string
+}
+
+type RepositoryPath struct {
+	Project    string `xml:"project,attr"`
+	Repository string `xml:"repository,attr"`
+}
+
+type Build struct {
+	Disable *Disabled `xml:"disable,omitempty"`
+}
+
+type Disabled struct{}
+
+type Publish struct {
+	Disable *Disabled `xml:"disable,omitempty"`
+}
+
+type DebugInfo struct {
+	Disable *Disabled `xml:"disable,omitempty"`
+}
+
+type UseForBuild struct {
+	Disable *Disabled `xml:"disable,omitempty"`
+}
+
+type Person struct {
+	UserID string `xml:"userid,omitempty,attr"`
+	Role   Role   `xml:"Role,omitempty,attr"`
+}
+
+type Repository struct {
+	Repository     string            `xml:"name,attr"`
+	Architecture   []string          `xml:"arch"`
+	ReleaseTargets *[]ReleaseTarget  `xml:"releasetarget,omitempty"`
+	Paths          *[]RepositoryPath `xml:"path,omitempty"`
+}
+
+type ReleaseTarget struct {
+	ProjectName string `xml:"project,attr"`
+	Repository  string `xml:"repository,attr"`
+	Trigger     string `xml:"trigger,attr"`
+}
+
+type Role string
+
+const (
+	RoleBugOwner   Role = "bugowner"
+	RoleMaintainer Role = "maintainer"
+	RoleReviewer   Role = "reviewer"
+	RoleDownloader Role = "downloader"
+	RoleReader     Role = "reader"
+)
```

Comment:
Let's call this type PersonRole and move type definitions close to the Person struct